BUILD.gn

```diff
@@ -13,11 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/build.gni")
```

Comment:
Seems to also affect Android/Chromium.  Has this been compiled for each?

Reply:
I haven't tested those builds manually.  Is there a documented process I can follow?

Reply:
Not really. I asked @mikes-lunarg to run this through the ANGLE build, which should do enough testing for us. I'll let him say how it works out.

Reply:
Sounds like @mikes-lunarg can help test the GN build.